tests/phpunit/tests/compat/str_ends_with.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @group compat
+ *
+ * @covers ::str_ends_with
+ */
+class Tests_Compat_str_ends_with extends WP_UnitTestCase {
+
+	/**
+	 * Test that is_iterable() is always available (either from PHP or WP).
+	 *
+	 * @ticket 43619
+	 */
+	public function test_is_iterable_availability() {
+		$this->assertTrue( function_exists( 'str_ends_with' ) );
+	}
```

Comment:
I think this is another copy-paste error.  Shouldn't the `@ticket` tag be `54377` and the DocBlock and function name mention `str_starts_with()` instead of `is_iterable()` :-?